### PR TITLE
[Compute AG: CWP-36493]: Email alerts troubleshooting update

### PR DESCRIPTION
**Pull request by divyabhushan-okta**
## Reference
[CWP-36493](https://redlock.atlassian.net/browse/CWP-36493)

### CODE REVIEW

File: compute/admin_guide/alerts/email.adoc [hunk 1]
```diff
@@ -51,23 +51,56 @@ include::frag_config_triggers.adoc[leveloffset=0]
...
 . Review the *Summary* and test the configuration by selecting *Send test alert*.
 
-. Click *Save*.
+. Select *Save*.
+
+=== Troubleshooting
+
+[.task]
+
+==== Unable to test Email Alerts for 'smtp.office365.com' on port 587.
```
Comment (Pubs-MV) [inline compute/admin_guide/alerts/email.adoc:81]: @divyabhushan can you please confirm if this topic is in PCCE and PCEE AG?

If yes, is this section for PCCE only?

Comment (divyabhushan-okta) [reply compute/admin_guide/alerts/email.adoc:81]: @Pubs-MV Yes, this topic and section is for both PCCE and PCEE. Thanks

Comment (Pubs-MV) [reply compute/admin_guide/alerts/email.adoc:81]: Can you please confirm this with QA once.

Comment (divyabhushan-okta) [reply compute/admin_guide/alerts/email.adoc:81]: @Pubs-MV Confirmed this information on the ticket.

**Pull request merged**